Update develop-tables-external-tables.md

charithcaldera opened the pull request:
Document update is required since its not mentioned about "GRANT Database Scoped Credential Permissions" in Synapse Serverless SQL Pool.

charithcaldera committed:
Update develop-tables-external-tables.md

Document update is required since its not mentioned about "GRANT Database Scoped Credential Permissions" in Synapse Serverless SQL Pool.

jovanpop-msft commented on articles/synapse-analytics/sql/develop-tables-external-tables.md:
````diff
@@ -181,6 +181,8 @@ CREATE EXTERNAL DATA SOURCE SqlOnDemandDemo WITH (
 );
 ```
+> [!NOTE]
+> The SQL users needs to have proper permissions on database scoped credentials to access the data source in Azure Synapse Analytics Serverless SQL Pool. [GRANT Database Scoped Credential Permissions](https://docs.microsoft.com/en-us/sql/t-sql/statements/grant-database-scoped-credential-transact-sql?view=sql-server-ver15&viewFallbackFrom=azure-sqldw-latest).
````
You might reference this page: https://docs.microsoft.com/en-us/azure/synapse-analytics/sql/develop-storage-files-overview?tabs=impersonation#permissions

azaricstefan replied:
I agree with @jovanpop-msft this page has more relevant explanations.
Also remove en-us from the url when modifying due to localization.

charithcaldera replied:
Changed applied

charithcaldera committed:
Update develop-tables-external-tables.md

Changes applied 
Link : https://docs.microsoft.com/azure/synapse-analytics/sql/develop-storage-files-overview?tabs=impersonation#permissions
